docs/_coverpage.md

```diff
@@ -10,5 +10,6 @@
 [Get Started](README.md)
 
 [See what's new in 3.0!](version_3.md)
+[Try in Playground](https://www.style-dictionary-play.dev/)
```

Comment:
Could we remove this from the cover page? I don't think we are quite ready for that placement.

Reply:
Can imagine ;) worth a try! I removed it